help/search/_posts/2015-02-09-search-syntax.md

```diff
@@ -169,7 +186,7 @@ is the search `object:"16 +60:0.1"`, followed by exporting the results in the `V
 
 ### Available Fields
 
-This is a list of fields currently recognized by the ADS search engine and the preferred search format:
+This is a list of fields currently recognized by the ADS search engine and the preferred search format - go to [comprehensive list of fields](./2016-07-29-comprehensive-solr-term-list.md) if not saturated yet:
```

Comment:
@kelockhart I've added the properties to search syntax and linked from here; for some however I don't know what should be there as a description; please edit as you see fit

also: IMHO the `adsopenaccess` is much better because it avoids any doubts where the break actually is (was it `ads_open_access`, `ads_openaccess`?) and it is a list of attributes, which just looks better as list of tokens